StyleCop.Analyzers/StyleCop.Analyzers/build/StyleCop.Analyzers.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)' == 'true'">
```

Comment:
This would need to occur unconditionally, since command line builds still need to be aware of the **stylecop.json** file in order to produce accurate results.